src/directives/repeat.ts

```diff
@@ -41,67 +96,139 @@ export function repeat<T>(
     keyFn = keyFnOrTemplate as KeyFn<T>;
   }
 
-  return directive((part: NodePart): void => {
-    let keyMap = keyMapCache.get(part);
-    if (keyMap === undefined) {
-      keyMap = new Map();
-      keyMapCache.set(part, keyMap);
-    }
-    const container = part.startNode.parentNode as HTMLElement | ShadowRoot |
-        DocumentFragment;
-    let index = -1;
-    let currentMarker = part.startNode.nextSibling!;
+  return directive((directivePart: NodePart): void => {
```

Comment:
I'd make this name match the name you use for `parentPart` in `createPart` &`movePart`

Reply:
👍Done.